specification/columns/skupricedetails.md

```diff
@@ -0,0 +1,55 @@
+# SKU Price Details
+
+The **SKU Price Details** column represents a list of relevant properties shared by all resources having the same [**SKU Price ID**](#skupriceid). These properties enable a practitioner to more easily calculate metrics such as total units of a service when it is not directly billed in those units (e.g. cores) and thus enables FinOps capabilities such as unit economics. They also help a practitioner understand the specifics of a **SKU Price ID**. 
```

Comment:
I believe this is about "charges" and not resources, given purchases will also have this and resources can have many SkuPriceId rows that may not be directly related to the resource, like bandwidth.
```suggestion
The **SKU Price Details** column represents a list of relevant properties shared by all charges with the same [**SKU Price ID**](#skupriceid). These properties enable practitioners to calculate metrics such as total units of a service when it is not directly billed in those units (e.g. cores) and thus enables FinOps capabilities such as unit economics. They also help a practitioner understand the specifics of a **SKU Price ID**.
```

Reply:
Thank you. I had reflected your terminology feedback in the normative section, but didn't in the introduction. I was looking for a better term for this [since the beginning](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/pull/503#issuecomment-2218426679), but you nailed it.